org.musescore.MuseScore.yaml

```diff
@@ -40,40 +36,19 @@ cleanup-commands:
 cleanup:
   - /share/man
 modules:
-  - name: portmidi
-    buildsystem: cmake-ninja
-    config-opts:
-      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
-      - -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=/app/lib
-      - -DCMAKE_ARCHIVE_OUTPUT_DIRECTORY=/app/lib
-      - -DCMAKE_RUNTIME_OUTPUT_DIRECTORY=/app/bin
-    post-install:
-      # MuseScore looks for this name for some reason
-      - ln -s libportmidi.so /app/lib/libporttime.so
-    sources:
-      - type: archive
-        url: https://sourceforge.net/projects/portmedia/files/portmidi/v2.0.2/portmidi-v2.0.2.zip
-        sha256: 322cbdfbb3af752ae46acd0134d17fe7c5c8b4a8e802486fd1ef46bb3d571a98
-    cleanup:
-      - /bin
-      - /lib/pkgconfig
-      - /include
-      - '*.a'
-      - '*.la'
-
   - name: musescore
     buildsystem: cmake-ninja
     builddir: true
     config-opts:
       - -DCMAKE_BUILD_TYPE=RelWithDebInfo
       - -DMUSE_APP_BUILD_MODE=release
-      - -DMUE_DOWNLOAD_SOUNDFONT=ON
       - -DMUSE_ENABLE_UNIT_TESTS=OFF
+      - -DMUSE_MODULE_VST=ON
+      - -DMUSE_MODULE_VST_VST3_SDK_PATH=vst3sdk
```

Comment:
Now this is probably wrong. What should I specify here so that it points to the folder downloaded under `sources` below?

Reply:
cmake believe this is relative to the build dir. :scream: